File: app/src/main/java/ar/com/wolox/android/example/ui/home/HomeActivity.java
package ar.com.wolox.android.example.ui.home;

import ar.com.wolox.android.R;
import ar.com.wolox.wolmo.core.activity.WolmoActivity;

/**
 * Home Activity
 */

public class HomeActivity extends WolmoActivity {

    @Override
    protected int layout() {
        return R.layout.activity_home;
    }

    @Override
    protected void init() {
        replaceFragment(R.id.activityHomeBaseContent, new HomeFragment());
    }
}

File: app/src/main/java/ar/com/wolox/android/example/ui/home/HomeActivity.kt
package ar.com.wolox.android.example.ui.home

import ar.com.wolox.android.R

import ar.com.wolox.wolmo.core.activity.WolmoActivity
import javax.inject.Inject

/**
 * Home Activity
 */

class HomeActivity @Inject constructor() : WolmoActivity() {

    @Inject internal lateinit var homeFragment: HomeFragment

    override fun layout(): Int {
        return R.layout.activity_home
    }

    override fun init() {
        replaceFragment(R.id.activityHomeBaseContent, homeFragment)
    }
}